### DISCUSSION TITLE
rpm installation on MacBook M2 Pro

### DISCUSSION THREAD

**Discussion by shahpa** [rpm-software-management/rpm#2684] [Q&A]
I am trying to install `rpm` on MacBook M2 Pro using `brew`. But it fails with the below error. Same command works fine on M1 chip version of MacBook. 
```
brew install rpm
rpm: Linux is required for this software.
attr: Linux is required for this software.
acl: Linux is required for this software.
elfutils: Linux is required for this software.
libcap: Linux is required for this software.
Error: rpm: Unsatisfied requirements failed this build.
```

**Comment by pmatilai** [comment]
That's not something the rpm upstream can help with, this would be a packaging matter in brew. How that all works or who maintains it there, we have no idea.
